config/settings.d/puppet.yml.example

```diff
@@ -41,16 +41,20 @@
 # If you want to override the puppet_user above just for mco commands
 #:mcollective_user: peadmin
 
-# URL of the puppet master itself for API requests
+# Override use of Puppet's API to list environments, by default it will use only if
+# environmentpath is given in puppet.conf, else will look for environments in puppet.conf
+# (Puppet versions prior to 4.0 only)
+#:puppet_use_environment_api: true
+
+# Puppet master version used
+#:puppet_version: 4.1
```

Comment:
This probably contradicts what I wrote on the puppetca.yml file, but this should probably have a default value, or at least a config migration to add one.  At the moment it doesn't work out of the box.